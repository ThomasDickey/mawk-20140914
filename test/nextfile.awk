# $MawkId: nextfile.awk,v 1.1 2012/06/27 13:53:45 tom Exp $
# Test-script for MAWK
###############################################################################
# copyright 2012, Thomas E. Dickey
#
# This is a source file for mawk, an implementation of
# the AWK programming language.
#
# Mawk is distributed without warranty under the terms of
# the GNU General Public License, version 2, 1991.
###############################################################################
	{printf "%s(%d):", FILENAME, FNR; print $0; }
/I/	{nextfile;}
