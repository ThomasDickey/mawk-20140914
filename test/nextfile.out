full-awk.dat(1):This has to be a small file to check if mawk handles write errors correctly 
full-awk.dat(2):even on a full disk. It has to be smaller than the write buffer of the 
mawktest.dat(1):
mawktest.dat(2):#include  <zmalloc.h>
mawktest.dat(3):
mawktest.dat(4):extern unsigned hash() ;
mawktest.dat(5):
mawktest.dat(6):/* An array A is a pointer to an array of struct array,
mawktest.dat(7):   which is two hash tables in one.  One for strings
mawktest.dat(8):   and one for doubles.
mawktest.dat(9):
mawktest.dat(10):   each array is of size A_HASH_PRIME.
